webui/react/src/utils/types.ts

```diff
@@ -80,9 +80,9 @@ export const runStateToLabel: {[key in RunState]: string} = {
   [RunState.Deleted]: 'Deleted',
   [RunState.Errored]: 'Errored',
   [RunState.Paused]: 'Paused',
-  [RunState.StoppingCanceled]: 'StoppingCanceled',
-  [RunState.StoppingCompleted]: 'StoppingCompleted',
-  [RunState.StoppingError]: 'StoppingErrored',
+  [RunState.StoppingCanceled]: 'Canceling',
```

Comment:
I don't exactly remember where (iotype?) but I think we want these to stay close to the backend. that's why we introduced `*ToLabel` helpers.